### ISSUE TITLE
Parseint challenge test issues

### ISSUE THREAD

**Issue by kprenesti** [freeCodeCamp/freeCodeCamp#12728]
Challenge [use-the-parseint-function](http://beta.freecodecamp.com/en/challenges/basic-javascript/use-the-parseint-function) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

The parameters in the test string visible to the user aren't changed.  
convertToInteger("56") should return a number
convertToInteger("56") should return 56
convertToInteger("56") should return 77  // 56 != 77.
convertToInteger("JamesBond") should return NaN
```js
    
    
function convertToInteger(str) {
return parseInt(str);
}
convertToInteger("56");

    ```

**Comment by systimotic** [comment]
@kprenesti Thanks for QA'ing and reporting this issue!
Since you reported it, you get dibs on fixing it if you want. 🙂 If you don't want to, please let us know and we'll open it up to the community. 👍

The tests for this challenge are [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json#L5647). The test is testing for 77, but the message wasn't updated properly. 
The JamesBond test is also broken, if you could fix that as well that would be great!
```js
!isNaN(convertToInteger(\"JamesBond\")); ,
```
should be 
```js
isNaN(convertToInteger("JamesBond")),
```

To get your local development environment set up, check out our [CONTRIBUTING.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md).
If you need help, please go to our [contributors chatroom](https://gitter.im/FreeCodeCamp/Contributors).

Happy coding! 😄

**Comment by dhcodes** [comment]
@kprenesti To add to what @systimotic has said, the test should be `assert.isNaN(convertToInteger("JamesBond"),`

because the `isNaN` test isn't working as should. I just submitted a PR that you can compare with #12737

**Comment by systimotic** [comment]
Thanks @dhcodes! I updated my message.

**Comment by HKuz** [comment]
Hi folks, I'm going to clean up the formatting/title capitalization/etc. for the last four challenges in the JS section (that are new). I'll fix this issue and #12727 as well.
